app/assets/javascripts/discourse/app/lib/header-outlet-height.js

```diff
@@ -0,0 +1,19 @@
+export default function() {
+  const outlets = [".above-site-header-outlet", ".below-site-header-outlet"];
+  // If these outlets have height they impact timeline and usercard positioning
+
+  let outletHeights = 0;
+
+  outlets.forEach(function(outletClass) {
+    if (document.querySelector(outletClass)) {
+      let outlet = document.querySelectorAll(outletClass);
+      for (var i = 0; i < outlet.length; i++) {
```

Comment:
is there any reason you don't use forEach here ? https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/forEach

if you absolutly want to use for, var should at least be replaced by let